spring-xd-dirt/src/main/resources/application.yml

```diff
@@ -93,6 +93,8 @@ xd:
   messageRateMonitoring:
     enabled: false
   messagebus:
+    local:
+      polling:                     1000
```

Comment:
I thought about it. Since it is the integer max, I kept it this way: `${xd.messagebus.local.queueSize: #{T(Integer).MAX_VALUE}}`